packages/components/src/components/audio/audio.ts

```diff
@@ -0,0 +1,539 @@
+import { css, html } from 'lit';
+import { customElement } from '../../../src/internal/register-custom-element';
+import { HasSlotController } from '../../internal/slot';
+import { LocalizeController } from '../../utilities/localize';
+import { property, query, state } from 'lit/decorators.js';
+import { Wave } from './wave';
+import cx from 'classix';
+import SolidElement from '../../internal/solid-element';
+import type SdDrawer from '../drawer/drawer';
+
+/**
+ * @summary Short summary of the component's intended use.
+ * @status experimental
+ * @since 1.0
+ *
+ * @dependency sd-icon
+ *
+ * @event sd-playback-start - Emitted when the audio playback starts.
+ * @event sd-playback-end - Emitted when the audio playback ends.
+ * @event sd-playback-pause - Emitted when the audio playback pauses.
+ * @event sd-playback-mute - Emitted when the audio is muted.
+ * @event sd-playback-unmute - Emitted when the audio is unmuted.
+ * @event sd-playback-speed - Emitted when the playback speed is changed.
+ * @event sd-transcript-click - Emitted when the transcript button is clicked.
+ *
+ * @slot - The default slot.
+ * @slot play-icon - The play icon.
+ * @slot pause-icon - The pause icon.
+ * @slot transcript - The transcript.
+ *
+ * @csspart base - The component's base wrapper.
+ * @csspart audio-controls - The audio controls.
+ * @csspart playback-speed - The playback speed.
+ * @csspart play-button - The play button.
+ * @csspart volume - The volume button.
+ * @csspart progress-slider - The audio progress slider.
+ * @csspart timestamps - The audio timestamps.
+ */
+@customElement('sd-audio')
+export default class SdAudio extends SolidElement {
+  private readonly localize = new LocalizeController(this);
+
+  private readonly hasSlotController = new HasSlotController(this, 'transcript');
+
+  /** Reverses the order of the audio controls and timestamps */
+  @property({ type: Boolean, reflect: true, attribute: 'reversed-layout' }) reversedLayout = false;
+
+  /** Shows or hides the timestamps */
+  @property({ type: Boolean, reflect: true, attribute: 'hide-timestamps' }) hideTimestamps = false;
+
+  /** Enables or disables the wave animation */
+  @property({ type: Boolean, reflect: true }) animated = false;
+
+  /** Inverts the colors of the component elements in darker backgrounds */
```

Comment:
```suggestion
  /** Inverts the colors of the component. */
```

It's not changing "on dark backgrounds".

Reply:
Updated